osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -36,6 +36,7 @@ public GlobalActionContainer(OsuGameBase game)
             new KeyBinding(new[] { InputKey.Control, InputKey.O }, GlobalAction.ToggleSettings),
             new KeyBinding(new[] { InputKey.Control, InputKey.D }, GlobalAction.ToggleDirect),
             new KeyBinding(new[] { InputKey.Control, InputKey.N }, GlobalAction.ToggleNotifications),
+            new KeyBinding(new[] { InputKey.Control, InputKey.A }, GlobalAction.ToggleNews),
```

Comment:
<kbd>Ctrl</kbd>+<kbd>A</kbd> seems like a pretty random binding, and I'm honestly not convinced it's going to get used.

Reply:
I thought that every overlay has a shortcut, now removed

Reply:
It also conflicts with select all from textboxes, so if it does get a keybind, it can't be this.